### ISSUE TITLE
[Issue]: [10.8.0] DLNA Play to / Remote Control not working for LG TV

### ISSUE THREAD

**Issue by xnick** [jellyfin/jellyfin#6922]
### Please describe your bug

I'm running the Alpha version through jellyfin-git on the AUR. Since yesterday I have been unable to remote control the TV via the jellyfin android app, or the browser UI. Playback from the TV itself works fine.

Disclaimer: this might actually be an LG issue, since I think my TV updated itself yesterday. But I'm hoping the logs might reveal something...

Thank you.

### Jellyfin Version

Other

### if other:

10.8.0 / Latest git commit / updated right before submission.

### Environment

```markdown
- OS: Arch Linux
- Virtualization: No
- Clients: LGTV WebOS (04.30.35) via DLNA
```

### Jellyfin logs

```shell
[2021-11-28 11:21:28.710 +00:00] [ERR] [54] Jellyfin.Server.Middleware.ExceptionMiddleware: Error processing request. URL "POST" "/Sessions/b02a2344c9f6f80fbc5c5835d3494e2d/Playing".
System.Net.Http.HttpRequestException: Response status code does not indicate success: 500 (Internal Server Error).
   at System.Net.Http.HttpResponseMessage.EnsureSuccessStatusCode()
   at Emby.Dlna.PlayTo.SsdpHttpClient.SendCommandAsync(String baseUrl, DeviceService service, String command, String postData, String header, CancellationToken cancellationToken) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Emby.Dlna/PlayTo/SsdpHttpClient.cs:line 46
   at Emby.Dlna.PlayTo.Device.SetAvTransport(String url, String header, String metaData, CancellationToken cancellationToken) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Emby.Dlna/PlayTo/Device.cs:line 346
   at Emby.Dlna.PlayTo.PlayToController.SetPlaylistIndex(Int32 index, CancellationToken cancellationToken) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Emby.Dlna/PlayTo/PlayToController.cs:line 654
   at Emby.Dlna.PlayTo.PlayToController.PlayItems(IEnumerable`1 items, CancellationToken cancellationToken) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Emby.Dlna/PlayTo/PlayToController.cs:line 638
   at Emby.Server.Implementations.Session.SessionManager.SendMessageToSession[T](SessionInfo session, SessionMessageType name, T data, CancellationToken cancellationToken) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Emby.Server.Implementations/Session/SessionManager.cs:line 1096
   at Emby.Server.Implementations.Session.SessionManager.SendPlayCommand(String controllingSessionId, String sessionId, PlayRequest command, CancellationToken cancellationToken) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Emby.Server.Implementations/Session/SessionManager.cs:line 1200
   at Jellyfin.Api.Controllers.SessionController.Play(String sessionId, PlayCommand playCommand, Guid[] itemIds, Nullable`1 startPositionTicks, String mediaSourceId, Nullable`1 audioStreamIndex, Nullable`1 subtitleStreamIndex, Nullable`1 startIndex) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Api/Controllers/SessionController.cs:line 188
   at lambda_method1227(Closure , Object )
   at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|20_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at Jellyfin.Server.Middleware.ServerStartupMessageMiddleware.Invoke(HttpContext httpContext, IServerApplicationHost serverApplicationHost, ILocalizationManager localizationManager) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Server/Middleware/ServerStartupMessageMiddleware.cs:line 49
   at Jellyfin.Server.Middleware.WebSocketHandlerMiddleware.Invoke(HttpContext httpContext, IWebSocketManager webSocketManager) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Server/Middleware/WebSocketHandlerMiddleware.cs:line 38
   at Jellyfin.Server.Middleware.IpBasedAccessValidationMiddleware.Invoke(HttpContext httpContext, INetworkManager networkManager) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Server/Middleware/IpBasedAccessValidationMiddleware.cs:line 48
   at Jellyfin.Server.Middleware.LanFilteringMiddleware.Invoke(HttpContext httpContext, INetworkManager networkManager, IServerConfigurationManager serverConfigurationManager) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Server/Middleware/LanFilteringMiddleware.cs:line 43
   at Microsoft.AspNetCore.Authorization.Policy.AuthorizationMiddlewareResultHandler.HandleAsync(RequestDelegate next, HttpContext context, AuthorizationPolicy policy, PolicyAuthorizationResult authorizeResult)
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Jellyfin.Server.Middleware.QueryStringDecodingMiddleware.Invoke(HttpContext httpContext) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Server/Middleware/QueryStringDecodingMiddleware.cs:line 37
   at Swashbuckle.AspNetCore.ReDoc.ReDocMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.SwaggerUI.SwaggerUIMiddleware.Invoke(HttpContext httpContext)
   at Swashbuckle.AspNetCore.Swagger.SwaggerMiddleware.Invoke(HttpContext httpContext, ISwaggerProvider swaggerProvider)
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Jellyfin.Server.Middleware.RobotsRedirectionMiddleware.Invoke(HttpContext httpContext) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Server/Middleware/RobotsRedirectionMiddleware.cs:line 45
   at Jellyfin.Server.Middleware.LegacyEmbyRouteRewriteMiddleware.Invoke(HttpContext httpContext) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Server/Middleware/LegacyEmbyRouteRewriteMiddleware.cs:line 52
   at Microsoft.AspNetCore.ResponseCompression.ResponseCompressionMiddleware.InvokeCore(HttpContext context)
   at Jellyfin.Server.Middleware.ResponseTimeMiddleware.Invoke(HttpContext context) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Server/Middleware/ResponseTimeMiddleware.cs:line 64
   at Jellyfin.Server.Middleware.ExceptionMiddleware.Invoke(HttpContext context) in /home/xnick/.cache/yay/jellyfin-git/src/jellyfin/Jellyfin.Server/Middleware/ExceptionMiddleware.cs:line 101
```

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by xnick** [comment]
As it turns out, it was indeed an LG issue. The problem was fixed by another update today.

Sorry for polluting the issues. 🙃

**Comment by Zen3515** [comment]
I know this may not be the best place to ask.

I'm having a really bad time setting up the DLNA play to, it doesn't work for any of my tv.
Could you share your DLNA profile for the LG TV please? 
I'm using LG C1 oled.

**Comment by xnick** [comment]
What issue are you having? Is the TV not showing up? Are you connecting to the TV but can't control it? Does playback from the TV itself work? 

Anyway, I'm attaching my config for the TV, put it in `jellyfin/config/dlna/user/`. I only have the one at my home, so I just used a catch-all for the identifier.
[Home LG.xml.txt](https://github.com/jellyfin/jellyfin/files/7779985/Home.LG.xml.txt)

It generally works fine when not transcoding, so i try to keep my media in formats that can play natively. The most common culprit is DTS audio, which I turn into ac3. 

Transcoding does work, but controlling the stream can be hit or miss (seeking etc). And if for some reason it breaks, then it simply won't let you play the file again, not from where you left off, not from the start, nada.

**Comment by aplatypus** [comment]
I have an older LG smart TV (WebOS), as far as I know there are no further WebOS updates for this model.  Last time I looked, the response showed newer versions but none were available for out TV.  The appliance works fine.  

When I use Jellyfin with the LG TV as DLNA device, it reports "`This file is invalid.'" for most of the `.mkv` files I have on my server -- Flatpak Jellyfin Server installed.  One or two `.mkv` files actually _do_ work.  The remote control works well as long as I **_stay_** on the PLAY page for on th browser or the (`flatpak`) Jellyfin client.

I've found if I move from the play page/panel to  browse or look at other menus I can't get back to the screen for the "currently playing"  track.  That seems to be a UI issue, when using a browser I leave the "playing" tab open and make a new tab for looking at menus, config, etc.  May be that information will be a reasonable workaround(???)

Nice to know how to deal with all the MKV files that don't play (`This file is invalid.')
